app/index.js

```diff
@@ -50,6 +49,7 @@ const config = require('./config');
 
 config.init();
 
+const createMenu = require('./menu');
```

Comment:
Actually there is a need for this change, the createMenu file now relies on configuration options, therefore must be required after the config is initialised.